tg/decorators.py

```diff
@@ -329,7 +329,42 @@ class decode_params(object):
 
     By default the arguments are parsed in **JSON** format (which is
     currently the only supported format).
-
+    
+    This should be used like:
+        Image you are posting a payload of type 'application/json' like:
+        
+        ticket = {
```

Comment:
This should be formatted as code too, you probably want two `::` in the previous chapter and indent accordingly.